compiler/desugared/from_surface.ml

```diff
@@ -1395,6 +1398,7 @@ let check_unlabeled_exception
         Message.error ~pos:(Mark.get item)
           ~pos_msg:(fun ppf -> Format.pp_print_text ppf "Ambiguous exception")
           ~extra_pos:(List.map (fun p -> "Candidate definition", p) pos)
+          "%a" Format.pp_print_text
```

Comment:
What's the use of this?

Reply:
`pp_print_text` enables reformatting, e.g. turning all spaces into `@ ` which allows word-wrapping. This is much nicer than putting `@ ` everywhere in the message by hand, but unfortunately only works as long as the message is a pure string without formatting instructions.

It's a little ugly though, it could be nicer to have a flag for this ; but we're already juggling with optional and extensible arguments so I'm not sure it can be done in a reasonable way (unless of course we have a separate function `error_str`...)